### PR TITLE
Replace worldgenoverride.lua with leveldataoverride.lua

### PR DESCRIPTION
**Pull request by thasmo**
That should do it. 🤷‍♂️

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -227,9 +227,9 @@ Defines some pre-configured world settings for the server.
 - COMPLETE_DARKNESS
 - DST_CAVE
 
-**WORLD_OVERRIDES**  
-Sets the overrides-configuration for world generation. Basically it's just the content for the
-`worldgenoverride.lua` file. As this value can be pretty large it's recommended to put the
+**LEVELDATA_OVERRIDES**
```
Comment (BraisGabin) [inline docs/configuration.md:230]: You changued the variable name in the docs but the code doesn't use `LEVELDATA_OVERRIDES` it continues using `WORLD_OVERRIDES`.

Probably you must find all the ocurrences of `WORLD_OVERRIDES` and renamed. This is a Breaking Change... but Klei introduced it so I think that it's ok to do it.

**Review by BraisGabin — CHANGES_REQUESTED** [review]

File: docs/configuration.md (hunk 1 above)
Comment (thasmo) [reply docs/configuration.md:230]: Oh, indeed. Not my brightest hour. :D Gonna fix it. Thanks for the check!

Comment (thasmo) [reply docs/configuration.md:230]: Changed it. Also, we still don't have a final release yet, so breaking changes for minor versions are fine for now, I believe.

**Review by BraisGabin — APPROVED** [review]

**Review by BraisGabin — APPROVED** [review]
LGTM

**Pull request merged**